tests/Unit/App/Action/Backend/Account/UpdateAccountActionTest.php

```diff
@@ -94,19 +95,99 @@ function executeUpdateAccount(
 
     $firstShop = AccountSettings::getAccount()->shops->first();
 
-    expect(
-        $firstShop->carriers->pluck('externalIdentifier')
-            ->toArray()
-    )
+    $externalIdentifiers = $firstShop->carriers
+        ->pluck('externalIdentifier')
+        ->all();
+
+    expect($externalIdentifiers)
         ->toBe([
             'cheapcargo',
             'bol.com',
             'ups',
             'dhlforyou',
             'postnl',
-            'postnl:23991',
             'dhlparcelconnect',
             'dhleuroplus',
             'dhlforyou:12424',
         ]);
```

Comment:
1. Wordt het contract id alleen weggehaald bij postnl? waarom niet ook bij dhlforyou?
2. Wordt het contract id allleen weggehaald als een carrier meerdere malen voorkomt? e.g. `postnl`, `postnl:23991`

Reply:
En vanuit de code en functionaliteit zou ik verwachten dat juist de postnl _met_ contractid bewaard blijft, of is dit ergens anders / een andere test? Dan snap ik het niet :-(

Reply:
ik heb expres de contract postnl weggehaald uit de default response, zodat het default gedrag ook het meest standaard is, en er zo min mogelijk andere tests door geraakt worden